data/35e/wizards_of_the_coast/supplement/expanded_psionics_handbook/xph_abilitycategories.lst

```diff
@@ -40,3 +40,4 @@ ABILITYCATEGORY:Expanded Knowledge Feat		VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:N
 
 ABILITYCATEGORY:Psicrystal Personality		VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:YES	FRACTIONALPOOL:NO							CATEGORY:Special Ability	TYPE:PsicrystalPersonality										PLURAL:Psicrystal Personalities		DISPLAYLOCATION:Feats
 
+ABILITYCATEGORY:Kreen Subrace			VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:YES	FRACTIONALPOOL:NO		CATEGORY:Internal			TYPE:KreenSubrace											DISPLAYLOCATION:Race Traits
```

Comment:
Why is the race name shortened? Thr-Kreen Subrace would follow normal convention.